pkg/scheduler/testing/wrappers.go

```diff
@@ -383,6 +383,7 @@ func (p *PodWrapper) Req(resMap map[v1.ResourceName]string) *PodWrapper {
 	p.Spec.Containers = append(p.Spec.Containers, v1.Container{
 		Resources: v1.ResourceRequirements{
 			Requests: res,
+			Limits:   res,
```

Comment:
One thing I noticed here is that with these changes in the PodWrapper Req we would only able to create Guaranteed pods. What if we need to create pods of burstable/BE pods? Might be worth adding a ReqLimit method in the wrapper? This could be done in this PR or as a separate PR.

```
// ReqLimit adds a new container to the inner pod with given resource map.
func (p *PodWrapper) ReqLimit(request map[v1.ResourceName]string, limit map[v1.ResourceName]string) *PodWrapper {
	if len(resMap) == 0 {
		return p
	}
	req := v1.ResourceList{}
        lim := v1.ResourceList{}
	for k, v := range request {
		res[k] = resource.MustParse(v)
	}
	for k, v := range limit {
		lim[k] = resource.MustParse(v)
	}
	p.Spec.Containers = append(p.Spec.Containers, v1.Container{
		Resources: v1.ResourceRequirements{
			Requests: req,
			Limits:   lim,
		},
	})
	return p
}
```

Reply:
Probably yes it's a new helper, and it's better to keep Req as is to be able to create Burstable QoS too.

Reply:
Want to clarify that I am not suggesting to replace Req, rather recommending that we add another method (ReqLimit)